doc/classes/EditorContextMenuPlugin.xml

```diff
@@ -89,7 +89,26 @@
 			The "Create..." submenu of FileSystem dock's context menu. [method _popup_menu] and option callback will be called with list of paths of the currently selected files.
 		</constant>
 		<constant name="CONTEXT_SLOT_SCRIPT_EDITOR" value="2" enum="ContextMenuSlot">
-			Context menu of Scene dock. [method _popup_menu] will be called with the path to the currently edited script, while option callback will receive reference to that script.
+			Context menu of Script editor's script tabs. [method _popup_menu] will be called with the path to the currently edited script, while option callback will receive reference to that script.
+		</constant>
+		<constant name="CONTEXT_SLOT_SCRIPT_EDITOR_CODE" value="4" enum="ContextMenuSlot">
```

Comment:
Seems like `CONTEXT_SLOT_FILESYSTEM_CREATE` is not exposed, is that intended?

Reply:
It's higher in the list, they are out of order.
Apparently you can bind enums in a different order and affect how they appear in the list. I grouped them together, but maybe that wasn't a good idea 🤔

Reply:
Yeah I'd keep them in enum order to avoid having out of order numbers.